core/sequencer/server.go

```diff
@@ -255,8 +255,8 @@ func (s *Server) DefineRevisions(ctx context.Context,
 
 	// Query metadata about outstanding log items.
 	lastMeta, err := s.batcher.ReadBatch(ctx, in.DirectoryId, resp.HighestDefined)
-	if err != nil {
-		return nil, status.Errorf(codes.Internal, "ReadBatch(): %v", err)
+	if st := status.Convert(err); st.Code() != codes.OK {
```

Comment:
Is this a passing-by change? I can't see whether it was necessary for the purpose of this PR. But it makes things consistent, so probably it's fine.

Reply:
It's necessary because we now rely on "NotFound" to know that there are no more outstanding revisions.